CONTRIBUTIONS.md

```diff
@@ -36,9 +36,9 @@ The script requires a valid license of a [supported type](https://github.com/jfr
 
 With the script you can start one or two Artifactory instances using docker compose.
 
-The license is not supplied, but a [30 day trial license can be freely obtained](https://jfrog.com/start-free/#hosted) and will allow local development. Make sure the license saved as a multi line text file.
+The license is not supplied but is required for local development. Make sure the license saved as a multi line text file.
```

Comment:
We don't want it anymore?

Reply:
You can't get free license for a long time 😄